sktime/forecasting/ets.py

```diff
@@ -119,7 +119,8 @@ class AutoETS(_StatsModelsAdapter):
     return_params : bool, default=False
         Whether or not to return only the array of maximizing parameters.
     auto : bool, default=False
-        Set True to enable automatic model selection.
+        Set True to enable automatic model selection. If auto=True, then error,
+        trend, seasonal and trend_damped are ignored.
```

Comment:
Small typo: the argument name for AutoETS is `damped_trend`, not `trend_damped`.